applications/flight/Tasks/hw_monitor.py

```diff
@@ -15,3 +15,4 @@ async def main_task(self):
         alerts.set_value(self.debug, 'radio_available', cubesat.radio is not None)
         alerts.set_value(self.debug, 'rtc_available', cubesat.rtc is not None)
         alerts.set_value(self.debug, 'neopixel_available', cubesat.neopixel is not None)
+        alerts.set_value(self.debug, 'camera_available', cubesat.camera is not None and cubesat.sdcard is not None)
```

Comment:
probably want to split this into a camera/std card alert. This was we'd know which devices are actually offline (unless we genuinely cannot tell if the camera is available when the sdcard is not)